test/scenarios/periodics/rhel-9.2-cncf-conformance.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# Sourced from cleanup_scenario.sh and uses functions defined there.
+
+prepare_hosts() {
+    local primary_host_ip
+    primary_host_ip=$(cat "${SCENARIO_INFO_DIR}/${SCENARIO}/vms/host1/public_ip")
+    local secondary_host_ip
+    secondary_host_ip=$(cat "${SCENARIO_INFO_DIR}/${SCENARIO}/vms/host2/public_ip")
+    local primary_host_ssh_port
+    primary_host_ssh_port=$(cat "${SCENARIO_INFO_DIR}/${SCENARIO}/vms/host1/ssh_port")
+    local secondary_host_ssh_port
+    secondary_host_ssh_port=$(cat "${SCENARIO_INFO_DIR}/${SCENARIO}/vms/host2/ssh_port")
+    local primary_host_name
+    primary_host_name="$(full_vm_name host1)"
+    local secondary_host_name
+    secondary_host_name="$(full_vm_name host2)"
```

Comment:
All of these settings are read-only, so you can use "local -r var=value" and have declaration and definition together.

Reply:
Done.